docs/design-docs/eks/README.md

```diff
@@ -8,9 +8,16 @@ The Amazon Elastic Kubernetes Service (EKS) is the AWS service for deploying, ma
 #### Requirements:
 - AWS account with the IAM permissions listed on the [EKS module documentation](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/iam-permissions.md) - set credentials in `vars-secret.tf` file.
 - kubectl
-- terraform
+- terraform in version at least 0.12.9
+- helm and added repository: ``` helm repo add stable https://kubernetes-charts.storage.googleapis.com ```
+- awscli ``` aws configure ```
```

Comment:
AWS Credentials should be put in var-secrets.tf file, which template is included in this repo, so I suppose aws configure isn't mandatory.